README.md

```diff
@@ -225,6 +230,23 @@ limitations under the License.
 
 ## Release Notes
 
+### v1.2.0
+
+- added the ability to load JS files instead of just json files.
+    - if the file name extension is ".js", the node loader will use
```

Comment:
is this spec?
Is there any site where I can see the explanation regarding different usage between sync/async?

Reply:
At the top of the README in the "Usage" section, it explains the usage in sync and async mode.